master/internal/checkpoints/postgres_checkpoints.go

```diff
@@ -54,25 +54,40 @@ func GetModelIDsAssociatedWithCheckpoint(ctx context.Context, ckptUUID uuid.UUID
 	return modelIDs, nil
 }
 
+// ModelInfo is a struct containing info used for locating models.
+type ModelInfo struct {
+	ID      int
+	Version int
+	Name    string
+}
+
 // GetRegisteredCheckpoints gets the checkpoints in
 // the model registrys from the list of checkpoints provided.
-func GetRegisteredCheckpoints(ctx context.Context, checkpoints []uuid.UUID) (map[uuid.UUID]bool, error) {
+func GetRegisteredCheckpoints(ctx context.Context, checkpoints []uuid.UUID) (map[uuid.UUID]ModelInfo, error) {
```

Comment:
Has this change been accounted for in the relevant integration tests? I believe this is being used in the `postgres_checkpoints_intg_test.go`.

Do any adjustments need to be made to `postgres_experiments.go`? Or with the other use in `postgres_checkpoints.go` in `PatchCheckpoints`

Reply:
i think the runtime code previously just cared about membership in the map, not the values attached to it, so it should be fine.

Reply:
In `postgres_checkpoints_intg_test.go` it's checking the equality of the returned map with an expected map, where the expected map is a `map[uuid.UUID]bool`, I believe this test is failing due to this equality check not being adjusted.